web/templates/message_edit_notice_tooltip.hbs

```diff
@@ -1,7 +1,11 @@
 <div>
     <div>{{t "View edit history"}}</div>
     {{#if realm_allow_edit_history}}
```

Comment:
This preexisting check is in the wrong place. Looks like @sayamsamal introduced this in abece73c5a580e0bc6a27c7603974c1e281df6dd, which swapped the strings around without bringing the conditionals along.